### DISCUSSION TITLE
Replace a Broken Slingbox 500

### DISCUSSION THREAD

**Discussion by bill-lj** [GerryDazoo/Slinger#665] [Q&A]
I have been using a slingbox 500 using the slinger server.  I had retrieved the admin pw prior to sling media server shutdown.
Unfortunately, the physical device has failed.  
Would it be possible to obtain a replacement slingbox 500 on ebay and configure it to setup DHCP networking and use
my existing config file and remote definition files?  Should I  assume that the admin password that I retrieved was specifically for
my failed SB 500, and if so would admin/admin get the replacement box functioning?  
Additionally, I would not be able to block outgoing traffic, as my cable router doesn't provide that functionality.

**Comment by GerryDazoo** [comment]
You won't be able to use your old password. After a factory reset your replacement box should usable with the admin/admin credentials. Some people have had success installing their remote .bin file with SlingPlayer.
Most modern routers have the ability to block outgoing internet access look for Parental Controls. On my router it's under Security/Device filter.

**Comment by bill-lj** [comment]
Gerry,

As always, thank you so much for your insights.  If I could, I would like
to have just a bit more clarity.

- my cable provider supplies a router with extremely limited options,
basically I can define a port forwarding rule.
  There will be no option for me to block outgoing ports, if this is deal
breaker in your opinion, please let me know.
-I'm using your packages' executable slingbox_server.exe on a windows 10
platform
 the only mods that I have is to edit config.ini to change ipaddress and
password as well as an updated remote.txt file, and the remote.txt
 file will have the key mappings for my cable provider's remote.  The new
admin password will be admin after the hard reset (depressing
reset for 60 seconds).

So if I purchase a replacement SB 500 with a remote, connect and define
ethernet as DHCP, and go with config until it
tries to contact Sling for Config file and presumably times out:
- will I be able to run the slinger server executable and display
audio/visual from a steamer, e.g. Slinger_Player, VNC, SMPlayer,
  and connect to the remote via slinger ip address/Remote to execute my DVR
specific IR key codes?

Thanks

**Reply by GerryDazoo** [reply]
" There will be no option for me to block outgoing ports, if this is deal
breaker in your opinion, please let me know."
AFAIK Sling hasn't tried to brick 500s/M1s. But blocking the port outgoing is recommended just in case. What you can do if you've got an old cheap router laying around (doesn't everybody :-)) is connect that to your existing router and when you configure it's Internet connection define a static IP address but DO NOT configure a default gateway and connect your Slingbox to the cheap router. Without a default gateway configured the slingbox won't be able to reach anything outside the local LAN. On your vendors router set up a port forwarding rule to get packets to the slingbox.

Some people have reported being able to configure a custom bin file for the remote. For sure trying to configure a Sling supplied remote config will fail.
